From d9bfca577cb4c88d5ff5716c0dcffa1972629601 Mon Sep 17 00:00:00 2001
From: Ariel D'Alessandro <ariel.dalessandro@collabora.com>
Date: Fri, 4 Apr 2025 09:51:41 +0200
Subject: [PATCH] GENIO: board: Match upstream kernel device tree blob names

In order to boot upstream kernel, set the proper default dtb file paths.

Note upstream kernel support is an ongoing effort so, at the moment of
this commit, not all dtbs might be built but filenames are expected to
match properly.

Signed-off-by: Ariel D'Alessandro <ariel.dalessandro@collabora.com>
Signed-off-by: Louis-Alexis Eyraud <louisalexis.eyraud@collabora.com>
---
 configs/genio_1200_evk_defconfig          | 2 +-
 configs/genio_1200_evk_ufs_defconfig      | 2 +-
 configs/genio_1200_evk_ufs_qspi_defconfig | 2 +-
 configs/genio_350_evk_defconfig           | 2 +-
 configs/genio_510_evk_defconfig           | 2 +-
 configs/genio_510_evk_qspi_defconfig      | 2 +-
 configs/genio_700_evk_defconfig           | 2 +-
 configs/genio_700_evk_qspi_defconfig      | 2 +-
 configs/genio_720_evk_defconfig           | 2 +-
 configs/genio_720_evk_ufs_defconfig       | 2 +-
 configs/genio_720_evk_ufs_qspi_defconfig  | 2 +-
 11 files changed, 11 insertions(+), 11 deletions(-)

diff --git a/configs/genio_1200_evk_defconfig b/configs/genio_1200_evk_defconfig
index 61f2667827..0f3975604d 100644
--- a/configs/genio_1200_evk_defconfig
+++ b/configs/genio_1200_evk_defconfig
@@ -22,7 +22,7 @@ CONFIG_FIT=y
 CONFIG_FIT_SIGNATURE=y
 CONFIG_LEGACY_IMAGE_FORMAT=y
 # CONFIG_ARCH_FIXUP_FDT_MEMORY is not set
-CONFIG_DEFAULT_FDT_FILE="mt8395-genio-1200-evk"
+CONFIG_DEFAULT_FDT_FILE="mediatek/mt8395-genio-1200-evk"
 # CONFIG_DISPLAY_BOARDINFO is not set
 # CONFIG_CMD_CONSOLE is not set
 # CONFIG_CMD_BOOTD is not set
diff --git a/configs/genio_1200_evk_ufs_defconfig b/configs/genio_1200_evk_ufs_defconfig
index ff786a4018..5300c4fc95 100644
--- a/configs/genio_1200_evk_ufs_defconfig
+++ b/configs/genio_1200_evk_ufs_defconfig
@@ -22,7 +22,7 @@ CONFIG_FIT=y
 CONFIG_FIT_SIGNATURE=y
 CONFIG_LEGACY_IMAGE_FORMAT=y
 # CONFIG_ARCH_FIXUP_FDT_MEMORY is not set
-CONFIG_DEFAULT_FDT_FILE="mt8395-genio-1200-evk-ufs"
+CONFIG_DEFAULT_FDT_FILE="mediatek/mt8395-genio-1200-evk-ufs"
 # CONFIG_DISPLAY_BOARDINFO is not set
 # CONFIG_CMD_CONSOLE is not set
 # CONFIG_CMD_BOOTD is not set
diff --git a/configs/genio_1200_evk_ufs_qspi_defconfig b/configs/genio_1200_evk_ufs_qspi_defconfig
index 5d5cf82c85..8750ab6bf6 100644
--- a/configs/genio_1200_evk_ufs_qspi_defconfig
+++ b/configs/genio_1200_evk_ufs_qspi_defconfig
@@ -23,7 +23,7 @@ CONFIG_FIT=y
 CONFIG_FIT_SIGNATURE=y
 CONFIG_LEGACY_IMAGE_FORMAT=y
 # CONFIG_ARCH_FIXUP_FDT_MEMORY is not set
-CONFIG_DEFAULT_FDT_FILE="mt8395-genio-1200-evk-ufs"
+CONFIG_DEFAULT_FDT_FILE="mediatek/mt8395-genio-1200-evk-ufs"
 # CONFIG_DISPLAY_BOARDINFO is not set
 # CONFIG_CMD_CONSOLE is not set
 # CONFIG_CMD_BOOTD is not set
diff --git a/configs/genio_350_evk_defconfig b/configs/genio_350_evk_defconfig
index e8822ed11a..6ceef8e7f0 100644
--- a/configs/genio_350_evk_defconfig
+++ b/configs/genio_350_evk_defconfig
@@ -21,7 +21,7 @@ CONFIG_DISTRO_DEFAULTS=y
 CONFIG_FIT=y
 CONFIG_FIT_SIGNATURE=y
 # CONFIG_ARCH_FIXUP_FDT_MEMORY is not set
-CONFIG_DEFAULT_FDT_FILE="mt8365-evk"
+CONFIG_DEFAULT_FDT_FILE="mediatek/mt8365-evk"
 # CONFIG_DISPLAY_BOARDINFO is not set
 # CONFIG_CMD_CONSOLE is not set
 # CONFIG_CMD_BOOTD is not set
diff --git a/configs/genio_510_evk_defconfig b/configs/genio_510_evk_defconfig
index c618a6872c..dc26a3fd6d 100644
--- a/configs/genio_510_evk_defconfig
+++ b/configs/genio_510_evk_defconfig
@@ -22,7 +22,7 @@ CONFIG_FIT=y
 CONFIG_FIT_SIGNATURE=y
 CONFIG_LEGACY_IMAGE_FORMAT=y
 # CONFIG_ARCH_FIXUP_FDT_MEMORY is not set
-CONFIG_DEFAULT_FDT_FILE="mt8370-genio-510-evk"
+CONFIG_DEFAULT_FDT_FILE="mediatek/mt8370-genio-510-evk"
 # CONFIG_DISPLAY_BOARDINFO is not set
 # CONFIG_CMD_CONSOLE is not set
 # CONFIG_CMD_BOOTD is not set
diff --git a/configs/genio_510_evk_qspi_defconfig b/configs/genio_510_evk_qspi_defconfig
index 613bde4244..c6674a431a 100644
--- a/configs/genio_510_evk_qspi_defconfig
+++ b/configs/genio_510_evk_qspi_defconfig
@@ -23,7 +23,7 @@ CONFIG_FIT=y
 CONFIG_FIT_SIGNATURE=y
 CONFIG_LEGACY_IMAGE_FORMAT=y
 # CONFIG_ARCH_FIXUP_FDT_MEMORY is not set
-CONFIG_DEFAULT_FDT_FILE="mt8370-genio-510-evk"
+CONFIG_DEFAULT_FDT_FILE="mediatek/mt8370-genio-510-evk"
 # CONFIG_DISPLAY_BOARDINFO is not set
 # CONFIG_CMD_CONSOLE is not set
 # CONFIG_CMD_BOOTD is not set
diff --git a/configs/genio_700_evk_defconfig b/configs/genio_700_evk_defconfig
index 076b802d76..43d7144406 100644
--- a/configs/genio_700_evk_defconfig
+++ b/configs/genio_700_evk_defconfig
@@ -22,7 +22,7 @@ CONFIG_FIT=y
 CONFIG_FIT_SIGNATURE=y
 CONFIG_LEGACY_IMAGE_FORMAT=y
 # CONFIG_ARCH_FIXUP_FDT_MEMORY is not set
-CONFIG_DEFAULT_FDT_FILE="mt8390-genio-700-evk"
+CONFIG_DEFAULT_FDT_FILE="mediatek/mt8390-genio-700-evk"
 # CONFIG_DISPLAY_BOARDINFO is not set
 CONFIG_BOARD_LATE_INIT=y
 # CONFIG_CMD_CONSOLE is not set
diff --git a/configs/genio_700_evk_qspi_defconfig b/configs/genio_700_evk_qspi_defconfig
index a611cbfe5d..f9290b6957 100644
--- a/configs/genio_700_evk_qspi_defconfig
+++ b/configs/genio_700_evk_qspi_defconfig
@@ -23,7 +23,7 @@ CONFIG_FIT=y
 CONFIG_FIT_SIGNATURE=y
 CONFIG_LEGACY_IMAGE_FORMAT=y
 # CONFIG_ARCH_FIXUP_FDT_MEMORY is not set
-CONFIG_DEFAULT_FDT_FILE="mt8390-genio-700-evk"
+CONFIG_DEFAULT_FDT_FILE="mediatek/mt8390-genio-700-evk"
 # CONFIG_DISPLAY_BOARDINFO is not set
 CONFIG_BOARD_LATE_INIT=y
 # CONFIG_CMD_CONSOLE is not set
diff --git a/configs/genio_720_evk_defconfig b/configs/genio_720_evk_defconfig
index 0d850ecc54..b53b6f27b8 100644
--- a/configs/genio_720_evk_defconfig
+++ b/configs/genio_720_evk_defconfig
@@ -21,7 +21,7 @@ CONFIG_DISTRO_DEFAULTS=y
 CONFIG_FIT=y
 CONFIG_FIT_SIGNATURE=y
 CONFIG_OF_BOARD_SETUP=y
-CONFIG_DEFAULT_FDT_FILE="mt8391-genio-720-evk"
+CONFIG_DEFAULT_FDT_FILE="mediatek/mt8391-genio-720-evk"
 # CONFIG_DISPLAY_BOARDINFO is not set
 CONFIG_BOARD_LATE_INIT=y
 # CONFIG_CMD_CONSOLE is not set
diff --git a/configs/genio_720_evk_ufs_defconfig b/configs/genio_720_evk_ufs_defconfig
index dd9d993bfa..186992e501 100644
--- a/configs/genio_720_evk_ufs_defconfig
+++ b/configs/genio_720_evk_ufs_defconfig
@@ -21,7 +21,7 @@ CONFIG_DISTRO_DEFAULTS=y
 CONFIG_FIT=y
 CONFIG_FIT_SIGNATURE=y
 CONFIG_OF_BOARD_SETUP=y
-CONFIG_DEFAULT_FDT_FILE="mt8391-genio-720-evk-ufs"
+CONFIG_DEFAULT_FDT_FILE="mediatek/mt8391-genio-720-evk-ufs"
 # CONFIG_DISPLAY_BOARDINFO is not set
 CONFIG_BOARD_LATE_INIT=y
 # CONFIG_CMD_CONSOLE is not set
diff --git a/configs/genio_720_evk_ufs_qspi_defconfig b/configs/genio_720_evk_ufs_qspi_defconfig
index 5516603d9f..9c17e150b3 100644
--- a/configs/genio_720_evk_ufs_qspi_defconfig
+++ b/configs/genio_720_evk_ufs_qspi_defconfig
@@ -22,7 +22,7 @@ CONFIG_DISTRO_DEFAULTS=y
 CONFIG_FIT=y
 CONFIG_FIT_SIGNATURE=y
 CONFIG_OF_BOARD_SETUP=y
-CONFIG_DEFAULT_FDT_FILE="mt8391-genio-720-evk-ufs"
+CONFIG_DEFAULT_FDT_FILE="mediatek/mt8391-genio-720-evk-ufs"
 # CONFIG_DISPLAY_BOARDINFO is not set
 CONFIG_BOARD_LATE_INIT=y
 # CONFIG_CMD_CONSOLE is not set
